testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4032,6 +4032,25 @@ function getColsCountFromDOM($form) {
     return result;
 }
 
+QUnit.test('Setting "colCountByScreen" property of group with dot notation should update layout', function(assert) {
```

Comment:
I didn't see tests with testing changing `colCountByScreen` option for Form & Tabbed items in runtime.
<img width="258" alt="image" src="https://user-images.githubusercontent.com/15689080/230367106-a6eac001-bb7b-46f6-92fe-2bc7177528fe.png">
Are you checked these scenarios. May be these test are written and you may add permalink to demonstrate them. 
If no I think we should check these cases.

Reply:
yes it uses the same code and working, ok i'll add some test